epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyVisibilityTracker.java

```diff
@@ -100,6 +102,15 @@ public void setOnChangedEnabled(boolean enabled) {
     onChangedEnabled = enabled;
   }
 
+  /**
+   * Set the threshold of percentage visible area to identify the partial impression view state.
+   *
+   * @param threshold Percentage of visible area of an element. 0..100.
+   */
+  public void setPartialImpressionThreshold(int threshold) {
```

Comment:
We could add `@IntRange(from = 0, to = 100)` to the param and I wonder if we should even name it `thresholdPercentage`

Reply:
`thresholdPercentage` sounds good

Reply:
832f83d3fbb076502d94184d6361b722d2966bab